app/data/events.json

```diff
@@ -1,4 +1,40 @@
 [
+	{
+		"date": "2024-09-29T18:00:00.000-04:00",
+		"link": "https://www.eventbrite.com/e/boston-ts-club-vi-rise-of-the-squiggles-tickets-1014606444187",
```

Comment:
I am very upset this event is named after SW episode 9 and not episode 6 "Return of the Jedi".

Reply:
@astorije if you change it here I'll be happy to change it on Eventbrite 😄

Reply:
Return of the Squiggle?

Reply:
Let's do it.

Reply:
Done. Note that I went with singular, not plural. [There seems to be controversy](https://www.google.com/search?q=%22return+of+the+jedi%22+singular+or+plural&oq=%22return+of+the+jedi%22+singular+or+plural&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRigATIHCAMQIRigATIHCAQQIRigATIHCAUQIRirAtIBCDY0NTFqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8) however so I'm happy to change 😅